service-definitions.js

```diff
@@ -39,13 +39,7 @@ var services = {
         }
     },
     frontend: {
-        name: "fe-application-loader",
-        commands: {
-            up: 'npm start #aww-frontend',
-            start: 'npm start #aww-frontend',
-            stop: "ps ax | grep \\#aww-frontend | grep -v grep | awk '{print $1}' | xargs -I % kill %",
-            isRunning: "ps ax | grep \\#aww-frontend | grep -v grep"
-        }
+        name: "fe-application-loader"
```

Comment:
Please also revert this since it is specific to you. I'll fix it in one of the future commits so that it is handled in a better manner.